spyder/app/mainwindow.py

```diff
@@ -2352,53 +2346,87 @@ def add_to_toolbar(self, toolbar, widget):
-            <p><small>Most of the icons for the Spyder 2 theme come from the
-            Crystal Project (&copy; 2006-2007 Everaldo Coelho).
-            Other icons for that theme come from
-            <a href="http://p.yusukekamiyamane.com/">
-            Yusuke Kamiyamane</a> (all rights reserved) and from
-            <a href="http://www.oxygen-icons.org/">
-            The Oxygen icon theme</a></small>.
+
+            <p>Python {python_ver} {bitness}-bit | Qt {qt_ver} |
+            {qt_api} {qt_api_ver} | {os_name} {os_ver}
+
+            <small><p>Certain source files under other compatible permissive
+            licenses and/or originally by other authors.
+            Spyder 3 theme icons derived from
+            <a href="https://fontawesome.com/">Font Awesome</a> 4.7
+            (&copy; 2016 David Gandy; SIL OFL 1.1).
+            Most Spyder 2 theme icons sourced from the
+            <a href="https://www.everaldo.com">Crystal Project iconset</a>
+            (&copy; 2006-2007 Everaldo Coelho; LGPL 2.1+).
+            Other icons from
+            <a href="http://p.yusukekamiyamane.com/">Yusuke Kamiyamane</a>
+            (&copy; 2013 Yusuke Kamiyamane; CC-BY 3.0),
+            the <a href="http://www.famfamfam.com/lab/icons/silk/">FamFamFam
+            Silk icon set</a> 1.3 (&copy; 2006 Mark James; CC-BY 2.5), and
+            the <a href="https://www.kde.org/">KDE Oxygen icons</a>
+            (&copy; 2007 KDE Artists; LGPL 3.0+).</small>
+
+            <p>See the <a href="{github_url}/blob/master/NOTICE.txt">NOTICE</a>
+            file for full legal information.
             """
-            % (versions['spyder'], revlink, __project_url__, __trouble_url__,
-               __project_url__, __forum_url__, versions['python'],
-               versions['bitness'], versions['qt'], versions['qt_api'],
-               versions['qt_api_ver'], versions['system'])
+            .format(spyder_ver=versions['spyder'],
+                    revision=revlink,
+                    website_url=__website_url__,
+                    github_url=__project_url__,
+                    trouble_url=__trouble_url__,
+                    forum_url=__forum_url__,
+                    python_ver=versions['python'],
+                    bitness=versions['bitness'],
+                    qt_ver=versions['qt'],
+                    qt_api=versions['qt_api'],
+                    qt_api_ver=versions['qt_api_ver'],
+                    os_name=versions['system'],
+                    os_ver=versions['release'])
         )
         msgBox.setWindowTitle(_("About %s") % "Spyder")
         msgBox.setStandardButtons(QMessageBox.Ok)
         msgBox.setIconPixmap(APP_ICON.pixmap(QSize(64, 64)))
-        msgBox.setTextInteractionFlags(Qt.TextSelectableByMouse)
-        msgBox.exec_()
+        msgBox.setTextInteractionFlags(
+            Qt.LinksAccessibleByMouse | Qt.TextSelectableByMouse)
+        msgBox.setModal(False)

```

Comment:
I know we all agreed on making the `About Spyder` dialog not modal, but I think this is kind of a ninja change doing it so here. I think this should be part of a separate PR with its own associated Issue where this can be documented and discussed.

Several dialogs are modal in Spyder (`Dependencies`, `Check for updates`, `Report issue`, `Python path manager`, etc...). It is a little bit inconsistent to make the `About dialog` the only one not modal and so, I think this needs to be discussed/documented in an Issue.

Reply:
Thanks for bringing this up, and sorry about that. Actually, I only did so in the first place since it was extremely annoying having it be modal when I was trying to go back and forth testing the changes. Then, I found a bug in the dialog (as noted in the commit message), that the links no longer worked after your change in your recent PR making the text selectable, so I had to fix that so the link to the legal notices and author attributions would actually work, as is of course required for them to fulfill their legal purpose. After I'd already had to make changes I apparently left the modal flag there as well; at this point its been so long and so much I can't 100% remember if it was a conscious decision or not.

The dependencies dialog should definitely not be modal as well, since it serves the same function as the about dialog (displaying useful information). However, the others are more arguable. In any case, I can open a separate issue to discuss it; for now I'll just revert this change.

Reply:
> Then, I found a bug in the dialog (as noted in the commit message), that the links no longer worked after your change in your recent PR making the text selectable

Thanks for catching that!

Reply:
Done